Slim/Menu/BrowseLibrary/Releases.pm

```diff
@@ -162,10 +166,22 @@ sub _releases {
 		!$primaryReleaseTypes{$_};
 	} keys %releaseTypes);
 
+	my $limitList = sub {
+		my $list = shift;
+		my $name = shift;
+		my $albumCount = scalar @$list;
+		if ( $albumCount > LIST_LIMIT ) {
+			splice @$list, LIST_LIMIT;
+			$name .= ' (' . cstring($client, 'FIRST') . ' ' . LIST_LIMIT . ' ' . cstring($client, 'ALBUMS') .')';
+		}
+		return $list, $name;
+	};
```

Comment:
I usually prefer not to create closures unless it really simplifies things by re-using a lot of the variables. The only one missing here would be `$client`. Would you mind changing this to a regular sub?

Adding new strings sucks. But concatenating strings often doesn't work with language other than "yours". Of the three languages I know one would fail (French). I know this will hardly ever be used, but could you still create a new string? Thanks!